core/sr-primitives/src/generic/digest.rs

```diff
@@ -58,6 +58,39 @@ impl<Item> traits::Digest for Digest<Item> where
 	}
 }
 
+/*
```

Comment:
i guess so. i kept it around since the structure of the generic `DigestItem` is no longer visible in code. that new macro makes it looks as though it doesn't have any members. pretty bad for code comprehension, really.